src/cleanlab_codex/validator.py

```diff
@@ -296,6 +318,25 @@ def _remediate(self, *, query: str, metadata: dict[str, Any] | None = None) -> s
         codex_answer, _ = self._project.query(question=query, metadata=metadata)
         return codex_answer
 
+    def _maybe_rewrite_query(self, *, query: str, messages: list[dict[str, Any]]) -> str:
```

Comment:
This _maybe... prefix implies that we might get something different from the method, other than a string. Should the check for `self._tlm` be done by the caller?

Reply:
the maybe is supposed to suggest we might re-write the query or not